### PR TITLE
Owner and nominate fixes for OVM support

### PR DESCRIPTION
**Pull request by justinjmoses**
This upgrade ensures the nominate and accept ownership steps function on L2.

### CODE REVIEW

File: publish/src/commands/owner.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const owner = async ({
 	const web3 = new Web3(new Web3.providers.HttpProvider(providerUrl));
 
 	const code = await web3.eth.getCode(newOwner);
-	const isContract = code !== '0x';
+	const isContract = !useOvm && code !== '0x';
```
Comment (ajsantander) [inline publish/src/commands/owner.js:89]: So, if it has code, it's not considered a contract if its on Optimism?
Why?

**Review by ajsantander — CHANGES_REQUESTED** [review]

File: publish/src/commands/owner.js (hunk 1 above)
Comment (justinjmoses) [reply publish/src/commands/owner.js:89]: It basically a hacky way of saying don't use contract owners on L2 - which honestly won't stick around anyhow. I can't recall why this was necessary at the time though frankly - why was `getCode` not return `0x` when this was run?

Comment (ajsantander) [reply publish/src/commands/owner.js:89]: Sorry, totally confused with what you're saying here 😅 

What you're doing is basically saying that "contract accounts in Optimism are considered EOAs" which I think is wrong.

**Review by ajsantander — APPROVED** [review]

File: publish/src/commands/owner.js (hunk 1 above)
Comment (i-stam) [reply publish/src/commands/owner.js]: I talked to Kevin about this. All accounts are initially EOAs (i.e. no code), but they are turned into a contract (proxy to the smart wallet) after they send the first tx. I 'm working to address this on the new owner-multisig script.

**Pull request closed without merging**